pkg/config/compatibility.go

```diff
@@ -260,18 +261,18 @@ func tfGPUPackage(ver string, cuda string) (name string, cpuVersion string, err
 	return "", "", nil
 }
 
-func torchCPUPackage(ver string, goos string, goarch string) (name string, cpuVersion string, findLinks string, err error) {
+func torchCPUPackage(ver, goos, goarch string) (name, cpuVersion, findLinks, extraIndexURL string, err error) {
 	for _, compat := range TorchCompatibilityMatrix {
 		if compat.TorchVersion() == ver && compat.CUDA == nil {
-			return "torch", torchStripCPUSuffixForM1(compat.Torch, goos, goarch), compat.FindLinks, nil
+			return "torch", torchStripCPUSuffixForM1(compat.Torch, goos, goarch), compat.FindLinks, compat.ExtraIndexURL, nil
 		}
 	}
 
 	// Fall back to just installing default version. For older pytorch versions, they don't have any CPU versions.
-	return "torch", ver, "", nil
+	return "torch", ver, "", "", nil
```

Comment:
Not gonna block but this is definitely getting to the point where this should be a struct :)